frontend/src/components/pages/CampCreation/CampCreationFooter.tsx

```diff
@@ -79,7 +85,12 @@ const CampCreationFooter = ({
         height="48px"
         variant="primary"
         onClick={onNextStep}
-        disabled={!isCurrentStepCompleted}
```

Comment:
Now, the next button is never really _disabled_, but now instead looks disabled (with the change in background color) when some fields haven't been filled out. This was done so we still have access to the `onClick` function even when something hasn't been filled.